htdocs/js/ui/notebook_protection_controller.js

```diff
@@ -94,6 +97,24 @@ define(['angular'], function(angular) {
 
         //NOTEBOOK TAB METHODS
         /////////////////////////////////////////
+        $scope.getUsers = function() {
+            return $q(function(resolve, reject) {
+                rcloud.get_users()
+                .then(function(data) {
+
+                    $scope.$evalAsync(function() {
+                        $scope.allUsers = _.map(data, function(user) { return { text : user, value : user }; });
+                    });
+
+                    //timeout to ensure evalAsync has finished
+                    $timeout(function() {
+                        logger.clear();
+                        resolve();
```

Comment:
It's smells a bit doesn't it? Having said that, I took the easier option, and duplicated the issue from other areas of the code. Perhaps it's something that can be looked at in the future.